src/endpoint/s3/s3_errors.js

```diff
@@ -144,6 +144,11 @@ S3Error.InvalidArgument = Object.freeze({
     message: 'Invalid Argument',
     http_code: 400,
 });
+S3Error.MissingInputSerialization = Object.freeze({
+    code: 'MissingRequiredParameter',
+    message: 'InputSerialization is required. Please check the service documentation and try again.',
```

Comment:
@alphaprinz, does this error appear in the [link above](https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
)? (I saw it [here](https://docs.aws.amazon.com/AmazonS3/latest/API/API_InputSerialization.html)).

Reply:
Yes. This message is what AWS returns (for this bad request).

Reply:
I'm sure.

In this file the errors are separated:
1. `Errors documented in AWS error pages` - in the link http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html (currently you put the error there).
3. `Errors actually returned by AWS S3 although not documented`

I asked since I didn't see it in the link of Errors documented in AWS error pages.

Reply:
The error code MissingRequiredParameter is documented in AWS (https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html#SelectObjectContentErrorCodeList).
The message (spefically for SerlizationInput param) is copied from AWS response.
So as far as I understand the location is ok?

Reply:
Thanks, yes.